File: level1/6 - Structures avancées/5 - Type d'arbres.c
//coded by cybereagle2001
# include <stdio.h>

int main()
{
   int hauteur;
   int nbDeFolioles;
   scanf("%d%d",&hauteur,&nbDeFolioles);
   if ((hauteur<=5) && (nbDeFolioles>=8))
   {
      printf("Tinuviel");
   }
   else
   {
      if ((hauteur>=10) && (nbDeFolioles>=10))
      {
         printf("Calaelen");
      }
      else
      {
         if((hauteur<=8) && (nbDeFolioles<=5))
         {
            printf("Falarion");
         }
         else 
         {
            if((hauteur>=12) && (nbDeFolioles<=7))
            {
               printf("Dorthonion");
            }
         }
      }
   }
}

File: java/level 1/6- Le grand voyage/5- Type d'arbres.java
/** coded by cybereagle2001*/
import algorea.Scanner;
class Main
{
public static void main(String[] args)
{
  Scanner entrée = new Scanner(System.in);
  int hauteur = entrée.nextInt();
  int nbFolioles = entrée.nextInt();
  if(hauteur < 9)
  {
     if(nbFolioles > 6)
     {
        System.out.println("Tinuviel");
     }
     else
     {
        System.out.println("Falarion");
     }
  }
  else
  {
     if(nbFolioles < 8)
     {
        System.out.println("Dorthonion");
     }
     else
     {
        System.out.println("Calaelen");
     }
  }
}
}
